diff -rupN svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/include/sdk_events.h svn10499.orig/trunk/src/include/sdk_events.h
--- svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/include/sdk_events.h	2015-09-17 18:52:34.000000000 +0200
+++ svn10499.orig/trunk/src/include/sdk_events.h	2015-09-17 18:41:32.000000000 +0200
@@ -340,10 +340,6 @@ extern EVTIMPORT const wxEventType cbEVT
 #define EVT_WORKSPACE_CHANGED(fn) DECLARE_EVENT_TABLE_ENTRY( cbEVT_WORKSPACE_CHANGED, -1, -1, (wxObjectEventFunction)(wxEventFunction)(CodeBlocksEventFunction)&fn, (wxObject *) NULL ),
 extern EVTIMPORT const wxEventType cbEVT_WORKSPACE_LOADING_COMPLETE;
 #define EVT_WORKSPACE_LOADING_COMPLETE(fn) DECLARE_EVENT_TABLE_ENTRY( cbEVT_WORKSPACE_LOADING_COMPLETE, -1, -1, (wxObjectEventFunction)(wxEventFunction)(CodeBlocksEventFunction)&fn, (wxObject *) NULL ),
-extern EVTIMPORT const wxEventType cbEVT_WORKSPACE_CLOSE_BEGIN;
-#define EVT_WORKSPACE_CLOSE_BEGIN(fn) DECLARE_EVENT_TABLE_ENTRY( cbEVT_WORKSPACE_CLOSE_BEGIN, -1, -1, (wxObjectEventFunction)(wxEventFunction)(CodeBlocksEventFunction)&fn, (wxObject *) NULL ),
-extern EVTIMPORT const wxEventType cbEVT_WORKSPACE_CLOSE_COMPLETE;
-#define EVT_WORKSPACE_CLOSE_COMPLETE(fn) DECLARE_EVENT_TABLE_ENTRY( cbEVT_WORKSPACE_CLOSE_COMPLETE, -1, -1, (wxObjectEventFunction)(wxEventFunction)(CodeBlocksEventFunction)&fn, (wxObject *) NULL ),
 
 // build targets events
 extern EVTIMPORT const wxEventType cbEVT_BUILDTARGET_ADDED;
diff -rupN svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/sdk/projectmanager.cpp svn10499.orig/trunk/src/sdk/projectmanager.cpp
--- svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/sdk/projectmanager.cpp	2015-09-17 18:54:04.000000000 +0200
+++ svn10499.orig/trunk/src/sdk/projectmanager.cpp	2015-09-17 18:41:06.000000000 +0200
@@ -573,11 +573,6 @@ bool ProjectManager::CloseWorkspace()
 {
     bool result = false;
     m_IsClosingWorkspace = true;
-
-    // Fire-up cbEVT_WORKSPACE_CLOSE_BEGIN event
-    CodeBlocksEvent evt1(cbEVT_WORKSPACE_CLOSE_BEGIN);
-    Manager::Get()->GetPluginManager()->NotifyPlugins(evt1);
-
     if (m_pWorkspace)
     {
         if (!m_ui->QueryCloseWorkspace())
@@ -601,11 +596,6 @@ bool ProjectManager::CloseWorkspace()
     else
         result = CloseAllProjects(false);
     m_IsClosingWorkspace = false;
-
-    // Fire-up cbEVT_WORKSPACE_CLOSE_COMPLETE event
-    CodeBlocksEvent evt2(cbEVT_WORKSPACE_CLOSE_COMPLETE);
-    Manager::Get()->GetPluginManager()->NotifyPlugins(evt2);
-
     WorkspaceChanged();
     return result;
 }
diff -rupN svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/sdk/sdk_events.cpp svn10499.orig/trunk/src/sdk/sdk_events.cpp
--- svn10499.cbEVT_WORKSPACE_CLOSE/trunk/src/sdk/sdk_events.cpp	2015-09-17 18:52:55.000000000 +0200
+++ svn10499.orig/trunk/src/sdk/sdk_events.cpp	2015-09-17 18:41:06.000000000 +0200
@@ -110,8 +110,6 @@ const wxEventType cbEVT_PROJECT_RENAMED
 const wxEventType cbEVT_PROJECT_OPTIONS_CHANGED = wxNewEventType();
 const wxEventType cbEVT_WORKSPACE_CHANGED = wxNewEventType();
 const wxEventType cbEVT_WORKSPACE_LOADING_COMPLETE = wxNewEventType();
-const wxEventType cbEVT_WORKSPACE_CLOSE_BEGIN = wxNewEventType();
-const wxEventType cbEVT_WORKSPACE_CLOSE_COMPLETE = wxNewEventType();
 // build targets events
 const wxEventType cbEVT_BUILDTARGET_ADDED = wxNewEventType();
 const wxEventType cbEVT_BUILDTARGET_REMOVED = wxNewEventType();
